x-pack/plugins/task_manager/server/task_running/task_runner.ts

```diff
@@ -340,6 +360,45 @@ export class TaskManagerRunner implements TaskRunner {
     }
   }
 
+  private validateTaskParams() {
+    let error;
+    const { state, taskType, params, id } = this.instance.task;
+
+    try {
+      const paramsSchema = this.definition.paramsSchema;
+      if (paramsSchema) {
+        paramsSchema.validate(params);
+      }
+    } catch (err) {
+      this.logger.warn(`Task (${taskType}/${id}) has a validation error: ${err.message}`);
```

Comment:
nit: maybe say that this task will be skipped to the warning log?

Reply:
A skip warning message is also printed in rescheduleFailedRun, i added this here because it's not always a skip error.